### PR TITLE
Weather v1.0

### PR DESCRIPTION
**Pull request by misselliev**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,57 @@
...
+    }
+  });
+}
+
+input.addEventListener('keydown', (e) => {
+  if (e.key === 'Enter') {
+    let city = input.value;
+    const proxy = 'https://cors-anywhere.herokuapp.com/';
+    const api = `${proxy}api.openweathermap.org/data/2.5/weather?q=${city}&APPID=ed8cea7d8f1ace4966a60e851d68906e`;
+    async function getWeather(city) {
```
Comment (AdnanAfsari) [inline src/index.js]: use Arrow function

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,57 @@
...
+const info = document.getElementById('info');
+const location = document.getElementById('location');
+const event = document.getElementById('event');
+const tempBox = document.getElementById('temperature');
+const temp = document.getElementById('temp-num');
+const degree = document.getElementById('degree');
+const description = document.getElementById('description');
+const input = document.querySelector('input');
+
+function loadData(data) {
```
Comment (AdnanAfsari) [inline src/index.js]: use Arrow function

**Review by AdnanAfsari — CHANGES_REQUESTED** [review]
**Status: Changes Required**

Hello @misselliev ✋ 

Nice work but some small changes to get approval.

- [x] usage of arrow functions instead of the function keyword.

- [x] The application should make different requests for the Celsius data and the Fahrenheit data. Instead of using math to convert it to the other unit.

Please make those changes and submit for another review.

Happy Coding
Adnan Afsari

**Comment by misselliev** [comment]
Hi @AdnanAfsari thank you for your comments! Just a question on the temperature unit topic, if we already have the temperature data why do we need to do another request and fetch it again instead of the toggle on click?

**Comment by AdnanAfsari** [comment]
**Status: Approved**

Hello @misselliev ✋ 

The purpose of this project is to learn making network request.The more you practice the better you get.
if you check the project requirement you will find it there, Look at the screenshot below i taken from there.
Screenshot 2019-11-07 at 9 10 06 AM

Anyway As you have done everything right, I am approving this project at this moment but i want you work on this requirement and fulfill it 😄 .

Happy Coding
Adnan Afsari

**Pull request merged**